potassium/potassium.py

```diff
@@ -1,35 +1,39 @@
-import requests
 from flask import Flask, request, make_response, abort
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import JSONResponse
+from fastapi.routing import APIRoute
 from werkzeug.serving import make_server
+from uvicorn import run as uvicorn_run
 from threading import Thread, Lock
 import functools
 from queue import Queue, Full
 import traceback
 from termcolor import colored
 
 
-class Endpoint():
+class Endpoint:
     def __init__(self, type, func):
         self.type = type
         self.func = func
 
 
-class Request():
+class Requests:
```

Comment:
I assume this breaking change was to avoid naming conflicts with the fastapi Request, but we'd want to avoid any breaking changes when possible. Could instead 

`from fastapi import Request as FastRequest` or something like that

Reply:
fixed